katas/content/qubit/index.md

```diff
@@ -58,21 +58,21 @@ $$
 \alpha \cdot \begin{bmatrix} 1 \\ 0 \end{bmatrix} + \beta \cdot \begin{bmatrix} 0 \\ 1 \end{bmatrix}
 $$
 
-Because of this, qubit states $0$ and $1$ are known as basis states. These two vectors have two properties.
+Because of this, qubit states $0$ and $1$ are known as **basis states**. These two vectors have two properties.
 
 1. They are normalized.
 
-$$
-\langle \begin{bmatrix} 1 \\ 0 \end{bmatrix} , \begin{bmatrix} 1 \\ 0 \end{bmatrix} \rangle =
-\langle \begin{bmatrix} 0 \\ 1 \end{bmatrix} , \begin{bmatrix} 0 \\ 1 \end{bmatrix} \rangle = 1
-$$
+    $$
+    \langle \begin{bmatrix} 1 \\ 0 \end{bmatrix} , \begin{bmatrix} 1 \\ 0 \end{bmatrix} \rangle =
+    \langle \begin{bmatrix} 0 \\ 1 \end{bmatrix} , \begin{bmatrix} 0 \\ 1 \end{bmatrix} \rangle = 1
+    $$
 
-2. They are orthogonal to each other.
+1. They are orthogonal to each other.
```

Comment:
Do we have to use the same number for list items? It reads easier for the person working with Markdown if they're different